docs/pages/tutorial/screenshot.mdx

```diff
@@ -73,7 +73,7 @@ We'll use `react-native-view-shot` to allow the user to take a screenshot within
 {/* prettier-ignore */}
 ```tsx app/(tabs)/index.tsx|collapseHeight=440
 import { useState, useRef } from 'react';
-import { captureRef } from "react-native-view-shot";
+ /* @tutinfo import { captureRef } from "react-native-view-shot";/* @end */
```

Comment:
```suggestion
/* @tutinfo Import <CODE>captureRef</CODE> from <CODE>react-native-view-shot</CODE>. */import { captureRef } from 'react-native-view-shot';/* @end */
```

The original suggestion will hide the import statement. I've made a suggestion to be more descriptive so that it follows the pattern we often use in the tutorial.

Reply:
oh my bad! thank you for fixing it, didn't realize, will take into account if I find this problem again!